Source/Scene/Model.js

```diff
@@ -4388,7 +4388,7 @@ function applySkins(model) {
         computedJointMatrices[m]
       );
       if (defined(bindShapeMatrix)) {
-        // Optimization for when bind shape matrix is the identity.
+        // NOTE: bindShapeMatrix is glTF 1.0 only, removed in glTF 2.0.
```

Comment:
CC https://github.com/CesiumGS/gltf-pipeline/issues/403 - ultimately `bindShapeMatrix` should be removed in `upgradeVersion`.